autoscheduler/scraper/models/department.py

```diff
@@ -1,4 +1,9 @@
 from django.db import models
 
 class Department(models.Model):
-    """ Describe model here """
+    """ Data structure that stores the department tag and description """

```

Comment:
I know I said to include it previously, but I honestly don't think this needs a doc-string, as it's super explanatory. Previously I wanted to do it just cause pylint said to, but I think it's overkill in this case at the least. Thoughts @rconn478